templates/controllers/modals/submission/viewSubmissionMetadata.tpl

```diff
@@ -0,0 +1,28 @@
+{**
+ * lib/pkp/templates/controllers/modals/submission/viewSubmissionMetadata.tpl
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2003-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Display submission metadata.
+ *}
+
+<div id="viewSubmissionMetadata" class="">
+	<h3>{$title|escape}</h3>
+	{if $authors}<h4>{$authors|escape}</h4>{/if}
+	<div class="abstract">
+		{$abstract}
+	</div>
```

Comment:
For the template, I'd recommend just adding the `publication` and using that for the title/abstract:

```
{$publication->getLocalizedFullTitle()|strip_unsafe_html}
```

Reply:
PHP Fatal error:  Uncaught Error: Call to undefined method Publication::getAbstract()

Reply:
So with this guessing that it just needs to be added, both Publication::getAbstract() and Publication::getLocalizedAbstract()

I mean I am still trying to get used to having publication.inc.php and submission.inc.php around

Reply:
So, we're quietly moving away from all these helper methods. We only use a helper method when it is actually doing some work (ie - `getLocalizedFullTitle()` concatenates the title parts). Otherwise, just use `->getData('abstract')` or `->getLocalizedData('abstract')`.

Reply:
ah, sounds like a good idea. check.